server/index.js

```diff
@@ -47,6 +47,17 @@ http
                 res.end(resultHtml);
             });
         }
+
+        else if (req.url === '/images/image.png') {
```

Comment:
This code is not necessary. Remove it.
Create an image with black square and add it to the `server/images` folder.

Rewrite the code below in the following way:
```js
// add to the top of file
const url         = require('url');
const querystring = require('querystring');
....

const repositoryRoot = path.resolve(__dirname, '..');
const resourcePath   = path.join(repositoryRoot, req.url);
const content        = fs.existsSync(resourcePath) ? fs.readFileSync(resourcePath).toString() : '';
const contentType    = CONTENT_TYPES[path.extname(resourcePath)];
const queryStr       = url.parse(req.url).query;
const delay          = parseInt(querystring.parse(queryStr).delay || 0);

if (contentType)
    res.setHeader('content-type', contentType);

 setTimeout(() => {
   res.end(content);
 }, delay);
```

Reply:
Add the proposed code to the example.

Reply:
added